test/org/sagebionetworks/bridge/services/ConsentServiceMockTest.java

```diff
@@ -88,116 +106,99 @@
     @Mock
     private SubpopulationService subpopService;
     @Mock
+    private NotificationsService notificationsService;
+    @Mock
+    private S3Helper s3Helper;
+    @Mock
+    private UrlShortenerService urlShortenerService; 
+    @Mock
     private Subpopulation subpopulation;
+    @Mock
+    private StudyConsentView studyConsentView;
     @Captor
     private ArgumentCaptor<BasicEmailProvider> emailCaptor;
     @Captor
     private ArgumentCaptor<WithdrawConsentEmailProvider> withdrawalProviderCaptor;
+    @Captor
+    private ArgumentCaptor<SmsMessageProvider> smsProviderCaptor;
+    @Captor
+    private ArgumentCaptor<Account> accountCaptor;
     
     private String documentString;
-        
-    private CriteriaContext context;
-    private Study study;
-    private StudyParticipant participant;
-    private ConsentSignature consentSignature;
+
     private Account account;
     
     @Before
     public void before() throws IOException {
         documentString = IOUtils.toString(new FileInputStream("conf/study-defaults/consent-page.xhtml"));
-                
+        account = new GenericAccount();
+        
         consentService = new ConsentService();
         consentService.setAccountDao(accountDao);
         consentService.setSendMailService(sendMailService);
         consentService.setActivityEventService(activityEventService);
         consentService.setStudyConsentService(studyConsentService);
         consentService.setSubpopulationService(subpopService);
+        consentService.setS3Helper(s3Helper);
+        consentService.setUrlShortenerService(urlShortenerService);
+        consentService.setNotificationsService(notificationsService);
         consentService.setConsentTemplate(new ByteArrayResource((documentString).getBytes()));
-
+        
         study = TestUtils.getValidStudy(ConsentServiceMockTest.class);
-        study.setConsentNotificationEmailVerified(true);
-        study.setSignedConsentTemplate(new EmailTemplate("Subject", "Body", MimeType.HTML));
+        study.setConsentNotificationEmail(study.getConsentNotificationEmail());
+        study.setConsentNotificationEmailVerified(study.isConsentNotificationEmailVerified());
+        study.setSignedConsentTemplate(study.getSignedConsentTemplate());
+        study.setSignedConsentSmsTemplate(study.getSignedConsentSmsTemplate());
```

Comment:
Since we're creating a new study instance every test, these setter calls are redundant now.

Reply:
Yes. :-1 removed.